app/scripts/modules/amazon/src/loadBalancer/AmazonLoadBalancersTag.tsx

```diff
@@ -40,15 +44,15 @@ class LoadBalancerListItem extends React.Component<ILoadBalancerListItemProps> {
 }
 
 @autoBindMethods
-class LoadBalancerButton extends React.Component<ILoadBalancerListItemProps> {
+class LoadBalancerButton extends React.Component<ILoadBalancerSingleItemProps> {
   private onClick(e: React.MouseEvent<HTMLElement>): void {
     this.props.onItemClick(this.props.loadBalancer);
     e.nativeEvent.preventDefault(); // yay angular JQueryEvent still listening to the click event...
   }
 
   public render(): React.ReactElement<LoadBalancerButton> {
     return (
-      <Tooltip value={`Load Balancer: ${this.props.loadBalancer.name}`}>
+      <Tooltip value={`${this.props.label || 'Load Balancer'}: ${this.props.loadBalancer.name}`}>
```

Comment:
It's used by target groups too, so yes, the default is the same as one of the options.
And no, this is used for when there's only one load balancer or target group.